aiven/client/cli.py

```diff
@@ -4790,7 +4790,13 @@ def _card_get_stripe_token(
     @arg("--update-project", help="Assign card to project")
     def card__add(self) -> None:
         """Add a credit card"""
-        stripe_key = self.client.get_stripe_key()["stripe_key"]
+        self.print_boxed(
+            [
+                "Command is deprecated and will be removed from the CLI in the future.",
```

Comment:
We should add wich cmd should be used instead.

Reply:
`avn card add has been deprecated and will be removed in the next release. Please use avn organization --organization-id organization_id card create`.